sycl/test-e2e/SubGroup/load_store.cpp

```diff
@@ -3,7 +3,6 @@
 //
 // Missing __spirv_SubgroupBlockReadINTEL, __spirv_SubgroupBlockWriteINTEL on
```

Comment:
is it still relevant since test is enabled for amd now?

Reply:
This is no longer relevant. Have removed